sdk/monitor/azure-monitor-query/README.md

```diff
@@ -405,6 +405,9 @@ raw JSON response. For example:
 }
 ```
 
+#### Overcoming Log Analytics query size limitations
+If your query exceeds the service limits, you can refer to the large log query documentation to learn how to overcome them.
```

Comment:
it's likely hard to add a link right away without failing the pipeline (only possible to add a permalink to your fork), but as a follow-up PR, mind adding a link to the other doc?